### DISCUSSION TITLE
Workflows should only run on origin repo

### DISCUSSION THREAD

**Discussion by danielhjacobs** [mdn/mdn-community#627] [Content] [answered]
This probably shouldn't run on forks.

**Comment by cw118** [comment]
Hi, I think disabling the workflow manually on your fork should resolve this:

![disable-workflow](https://user-images.githubusercontent.com/87150472/164938751-d529388d-e113-474e-80f4-5a5b903a095f.png)

You can do so in the "Actions" tab of your fork by clicking on the "PR review companion" workflow, then clicking the three dot menu.

**Reply by danielhjacobs** [reply]
This works, but I'm hesitant to mark it as an answer because I think it would be better if that action were set up to not run on forks by default.

I think this can be done by adding     `if: github.repository == 'mdn/content'` to the yml file for that workflow.

**Comment by Rumyra** [comment]
@schalkneethling I actually think this is a good idea - I never disable workflows & get notifications for my branch - what do you think?

**Reply by schalkneethling** [reply]
Yes! Thank you for the suggestion @danielhjacobs. Let me see if that is indeed the way to do this. Perhaps GitHub has a special mechanism to ensure the workflows only run on the origin repository. If not, your suggestion would be the go-to(no pun intended :)).

**Reply by danielhjacobs** [reply]
I assume if there were a different mechanism, it would be mentioned here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository

Edit: https://github.com/github/feedback/discussions/9098 / https://github.com/actions/runner/issues/859

**Reply by schalkneethling** [reply]
Thank you @danielhjacobs, I will look into this further tomorrow.

**Comment by schalkneethling** [comment, marked as answer]
Issue created: https://github.com/mdn/workflows/issues/53

**Comment by pepelsbey** [comment]
Closing as answered/settled.
